.travis.yml

```diff
@@ -27,4 +27,11 @@ install:
   # Default bundler version is buggy
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then rvm use 2.2.1; gem uninstall bundler -v1.13.1; fi
 
-script: ./tools/travis.sh
+script:
+  - ./tools/travis.sh
+  # spellcheck
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+      nvm install 6.4.0 &&
+      npm install -g markdown-spellcheck@0.11.0 &&
+      mdspell "**/*.md" --ignore-numbers --ignore-acronyms --report;
+    fi
```

Comment:
The file must end with `new line`.
Open [code review](https://github.com/PowerShell/PowerShell/pull/2554/files/b6e551794a259dbfb40f9a5ef88393c00e9dff65) or or look above my first message. There you will see a red mark in end of the file.